lib/meilisearch/client.rb

```diff
@@ -23,9 +23,9 @@ def indexes(options = {})
     end
 
     # Usage:
-    # client.create_index('indexUID')
-    # client.create_index('indexUID', primaryKey: 'id')
-    def create_index(index_uid, options = {})
+    # client.create_index('indexUID', wait: true)
+    # client.create_index('indexUID', primaryKey: 'id', wait: true)
+    def create_index(index_uid, options = {}, wait: true)
```

Comment:
I think I made a mistake when I wrote the issue, so I'm sorry if I was not clear :/

This issue aims to turn the methods with the bang operators into normal ones by providing an option. So, this is what I expect:

```rb
# before
index = client.create_index!('books') # returns an index, not a task
task = client.create_index('books') # returns a task, not an index

# after
index = client.create_index('books', wait: true) # returns an index, not a task
task = client.create_index('books') # returns a task, not an index
```

Reply:
Yes, I too thoughts so...I should have clarified before committing changes.

Reply:
I have updated the same in the new commits!